datalad/crawler/nodes/tests/test_misc.py

```diff
@@ -471,4 +473,9 @@ def node(d):
 def test_debug():
     yield _test_debug, "About to run"
     yield _test_debug, "Ran node .* which yielded 1 times", ('after',)
-    yield _test_debug, "Ran node .* which yielded 0 times", ('empty',)
\ No newline at end of file
+    yield _test_debug, "Ran node .* which yielded 0 times", ('empty',)
+
+
+def test_fix_url():
+    eq_(list(fix_url({'url': 'http://myweb.fsu.edu/bgomez/GomezWilson_2006_JOP_Replication Material.zip'})),
+        [{'url': 'http://myweb.fsu.edu/bgomez/GomezWilson_2006_JOP_Replication%20Material.zip'}])
```

Comment:
come up with few more examples for some "corner" cases even though if you know that they must work correctly. e.g. what about the space at the end of url?